tests/benchmark/Dockerfile.in~

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google-appengine/python:latest
```

Comment:
Seems like this is a backup file? I'd suggest removing this from this PR (`git remove Dockerfile.in~`) and then adding all `*~` files to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).